tests/Forms/Container.values.BackedEnum.phpt

```diff
@@ -0,0 +1,44 @@
+<?php
```

Comment:
We would need to somehow skip this file on PHP 8.0.

Reply:
Any idea please?

Reply:
Simply add 
```
/**
 * @phpVersion 8.1
 */
```

Reply:
Thanks added